tests/bin/install-wc-tests.sh

```diff
@@ -10,20 +10,28 @@ DB_USER=$2
 DB_PASS=$3
 DB_HOST=${4-localhost}
 WP_VERSION=${5-latest}
-WC_VERSION=${6-"5.0.0"}
+WC_VERSION=${6-"7.4.1"}
 
-install_wc() {
-	git clone --depth=1 --branch=$WC_VERSION https://github.com/woocommerce/woocommerce.git /tmp/woocommerce
-	composer install -d /tmp/woocommerce
+export NVM_DIR="$HOME/.nvm"
+NVM_PATH=/opt/homebrew/opt/nvm/nvm.sh
+
+# Load nvm.
+. $NVM_PATH
```

Comment:
Instead of 
```
NVM_PATH=/opt/homebrew/opt/nvm/nvm.sh

# Load nvm.
. $NVM_PATH
```

What do you think about using the same lines in our `.bashrc`?
```
export NVM_DIR="$HOME/.nvm"
[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm
[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
```

I didn't install mine via homebrew and this is throwing an error during `install_wc()` locally.